corehq/blobs/s3db.py

```diff
@@ -78,9 +82,8 @@ def delete(self, *args, **kw):
     def copy_blob(self, content, info, bucket):
         self._s3_bucket(create=True)
         path = self.get_path(info.identifier, bucket)
-        osutil = OpenFileOSUtils()
-        transfer = S3Transfer(self.db.meta.client, osutil=osutil)
-        transfer.upload_file(content, self.s3_bucket_name, path)
+        self._s3_bucket().upload_fileobj(content, path)
```

Comment:
Notable difference in new `upload_fileobj` implementation in boto3 (v1.4.0): https://github.com/boto/s3transfer/issues/52